README.md

```diff
@@ -1,10 +1,8 @@
-# alfred3-workflow-CurrencyConvert
+# alfred4-workflow-CurrencyConvert
 
-This alfred3 workflow can convert currency conveniently.
+This alfred4 workflow can convert currency conveniently.
 
-**Notice: The Old versions(v1.4-) api has been out of service. Please update to v1.5+ to keep using currency conversion feature.**
-
-[Download](https://github.com/jin5354/alfred3-workflow-CurrencyConvert/releases)
+[Download](https://github.com/topazbarziv/alfred4-workflow-CurrencyConvert/releases)
```

Comment:
This works for @topazbarziv's fork, but shouldn't be included when merging to this repo.